extensions/wikia/ExactTargetUpdates/client/ExactTargetClient.class.php

```diff
@@ -143,6 +143,23 @@ public function updateUserEdits( $edits ) {
 		return $this->sendRequest( self::UPDATE_CALL, $request );
 	}
 
+	public function updateWiki( $cityId, array $cityData ) {
+		$request = ExactTargetRequestBuilder::getWikiUpdateBuilder()
+			->withCityData( $cityId, $cityData )
+			->build();
+
+		return $this->sendRequest( self::UPDATE_CALL, $request );
+	}
+
+	public function updateWikiCatMapping( $cityId, array $categories ) {
```

Comment:
Does it correspond to `ExactTargetUpdateCityCatMappingTask.php`? If so, perhaps we can remove that task class, too?